ballerina/tests/mysql_test_entities_generated_client.bal

```diff
@@ -152,30 +152,30 @@ public isolated client class SQLTestEntitiesClient {
     };
 
     public isolated function init() returns persist:Error? {
-        mysql:Client|error dbClient = new (host = host, user = user, password = password, database = database, port = port);
+        mysql:Client|error dbClient = new (host = mysql.host, user = mysql.user, password = mysql.password, database = mysql.database, port = mysql.port);
         if dbClient is error {
             return <persist:Error>error(dbClient.message());
         }
         self.dbClient = dbClient;
         self.persistClients = {
-            [ALL_TYPES] : check new (self.dbClient, self.metadata.get(ALL_TYPES)),
-            [STRING_ID_RECORD] : check new (self.dbClient, self.metadata.get(STRING_ID_RECORD)),
-            [INT_ID_RECORD] : check new (self.dbClient, self.metadata.get(INT_ID_RECORD)),
-            [FLOAT_ID_RECORD] : check new (self.dbClient, self.metadata.get(FLOAT_ID_RECORD)),
-            [DECIMAL_ID_RECORD] : check new (self.dbClient, self.metadata.get(DECIMAL_ID_RECORD)),
-            [BOOLEAN_ID_RECORD] : check new (self.dbClient, self.metadata.get(BOOLEAN_ID_RECORD)),
-            [COMPOSITE_ASSOCIATION_RECORD] : check new (self.dbClient, self.metadata.get(COMPOSITE_ASSOCIATION_RECORD)),
-            [ALL_TYPES_ID_RECORD] : check new (self.dbClient, self.metadata.get(ALL_TYPES_ID_RECORD))
+            [ALL_TYPES] : check new (self.dbClient, self.metadata.get(ALL_TYPES), MYSQL_SPECIFICS),
+            [STRING_ID_RECORD] : check new (self.dbClient, self.metadata.get(STRING_ID_RECORD), MYSQL_SPECIFICS),
+            [INT_ID_RECORD] : check new (self.dbClient, self.metadata.get(INT_ID_RECORD), MYSQL_SPECIFICS),
+            [FLOAT_ID_RECORD] : check new (self.dbClient, self.metadata.get(FLOAT_ID_RECORD), MYSQL_SPECIFICS),
+            [DECIMAL_ID_RECORD] : check new (self.dbClient, self.metadata.get(DECIMAL_ID_RECORD), MYSQL_SPECIFICS),
+            [BOOLEAN_ID_RECORD] : check new (self.dbClient, self.metadata.get(BOOLEAN_ID_RECORD), MYSQL_SPECIFICS),
+            [COMPOSITE_ASSOCIATION_RECORD] : check new (self.dbClient, self.metadata.get(COMPOSITE_ASSOCIATION_RECORD), MYSQL_SPECIFICS),
+            [ALL_TYPES_ID_RECORD] : check new (self.dbClient, self.metadata.get(ALL_TYPES_ID_RECORD), MYSQL_SPECIFICS)
```

Comment:
Could this lead to a breaking change? Can we default to `MYSQL_SPECIFICS` if there is no value specified

Reply:
Done in [1fecb93](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/14/commits/1fecb931d87ca19f7efbef2f808662c28e17db4c)